daemon/containerd/image_list.go

```diff
@@ -225,6 +225,8 @@ func (i *ImageService) imageSummary(ctx context.Context, img images.Image, platf
 	var best *ImageManifest
 	var bestPlatform ocispec.Platform
 
+	var mut sync.Mutex
```

Comment:
Ah, I was going to say "why don't we move it inside `i.WalkReachableIamgeManifests`/next to `msftSummary` but I see it's also guarding `countainersCount`.

Can you move this closer to `containersCount` and add a comment explaining that it guards access to `containersCount` and `msftSummary`?

Reply:
Done